npm/vue/examples/code-coverage/patches/@cypress+code-coverage+3.9.5.patch

```diff
@@ -7,7 +7,7 @@ index 31e00ee..0c56908 100644
    /** @type {string} Cypress run-time config has test files string pattern */
    // @ts-ignore
 -  const testFilePattern = config('testFiles')
-+  const testFilePattern = config('specPattern')
++  const testFilePattern = config(Cypress.testingType)['specPattern']
```

Comment:
Just curious, but could this be?
```suggestion
+  const testFilePattern = config(Cypress.testingType).specPattern
```

Reply:
Oh, this works, nice! Did not realize you access nested properties with `.` in TS. https://github.com/cypress-io/cypress/pull/19345/commits/d6c8b5eac6c670d795decbf95dc83a257fe02c0b